src/Utopia/Messaging/Adapters/Push/PushBullet.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\Push;
+
+use Exception;
+use GuzzleHttp\Client;
+use Utopia\Messaging\Adapters\Push as PushAdapter;
+use Utopia\Messaging\Messages\Push;
+
+class PushBullet extends PushAdapter
```

Comment:
Let's name this Pushbullet:

```suggestion
class Pushbullet extends PushAdapter
```